client/src/util/fetchFromAPI.js

```diff
@@ -22,4 +22,5 @@ export const queryAPI = async (options, toast) => {
   } else {
     toast.error(`Food item not saved.  Error: ${result.error}`);
   }
+  return response;
```

Comment:
Ideally `queryAPI` would return the `result` object, not the `response` object.  This also means that you'll have to set `result.error` inside of the `else` branch of the condition.

Reply:
thank you -updated